config/routes.rb

```diff
@@ -3,6 +3,9 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
```

Comment:
- [ ] It would be better if our `likes` route looks similar to the image below
![Screenshot from 2023-01-26 22-29-37](https://user-images.githubusercontent.com/46388113/214964988-9e8c1b55-fa1a-4e63-a8b9-01b43cb54b19.png)
We can achieve this by nesting the likes resources into the post resources

Reply:
Hello @addod19 I think for readability is better to use the way I did it because:
1. Posts are already nested from users (this would be a three-level layer and is a good practice to keep the nested elements low)
2. Likes do not require a create  view (it is a partial code that is directly rendered on the post's page)

Please let me know if should I modify it, also it would be great if you could give me a reference page to read more about your recommendation.

Reply:
Sure, you are right we need to avoid the 3 level deep nesting. How about just the post resource and the likes resource. Let us [check](https://guides.rubyonrails.org/routing.html) this link for more insights

Reply:
Ok, I'll avoid 3-level nesting for this project, thank you @addod19